### PR TITLE
Small UI updates (to be closer to design guidelines)

### PR DESCRIPTION
**Pull request by tdejoigny-ledger**

### CODE REVIEW

File: src/ui/menu_nbgl.c [hunk 1]
```diff
@@ -49,7 +50,10 @@ static bool nav_callback(uint8_t page, nbgl_pageContent_t* content) {
 }
 
 void ui_menu_about() {
-    nbgl_useCaseSettings(APPNAME, 0, 1, false, ui_menu_main, nav_callback, NULL);
+#define TOTAL_PAGE_NB        (1)
+#define INIT_PAGE_INDEX      (0)
+#define DISABLE_SUB_SETTINGS (false)
```
Comment (xchapron-ledger) [inline src/ui/menu_nbgl.c:55]: What do you think of:
```suggestion
#define IS_SUB_SETTINGS (false)
```

File: src/ui/menu_nbgl.c [hunk 2]
```diff
@@ -31,7 +31,8 @@ void app_quit(void) {
 }
 
 void ui_menu_main(void) {
-    nbgl_useCaseHome(APPNAME, &C_app_boilerplate_64px, NULL, false, ui_menu_about, app_quit);
+ #define SETTINGS_BUTTON_DISABLED (false)
```
Comment (xchapron-ledger) [inline src/ui/menu_nbgl.c:34]:
```suggestion
 #define DISPLAY_SETTINGS_BUTTON (false)
```
?

File: src/ui/nbgl_display_address.c [hunk 3]
```diff
@@ -66,14 +62,7 @@ static void review_choice(bool confirm) {
...
 static void continue_review(void) {
-    // Fill pairs
-    pairs[0].item = "BIP32 Path";
-    pairs[0].value = g_bip32_path;
-
-    pairList.nbMaxLinesForValue = 0;
-    pairList.nbPairs = 1;
-    pairList.pairs = pairs;
-    nbgl_useCaseAddressConfirmationExt(g_address, review_choice, &pairList);
+    nbgl_useCaseAddressConfirmation(g_address, review_choice);
```
Comment (xchapron-ledger) [inline src/ui/nbgl_display_address.c:65]: Should we remove it from bagl UI too?

File: src/ui/menu_nbgl.c (hunk 2 above)
Comment (tdejoigny-ledger) [reply src/ui/menu_nbgl.c:34]: not sure.
If I have 
`nbgl_useCaseHome(APPNAME, &C_app_boilerplate_64px, NULL, DISPLAY_SETTINGS_BUTTON, ui_menu_about, app_quit);`
I don't know if the settings is displayed or not, the `#define` value should not be considered (especially if this `#define` is included in a header file subsequently)

File: src/ui/menu_nbgl.c (hunk 1 above)
Comment (tdejoigny-ledger) [reply src/ui/menu_nbgl.c:55]: same comment

Comment (xchapron-ledger) [reply src/ui/menu_nbgl.c:55]: indeed!

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**